packages/tss-client/src/utils.ts

```diff
@@ -112,10 +112,11 @@ export function getTSSPubKey(dkgPubKey: PointHex, userSharePubKey: key, userTSSI
   return serverTerm.add(userTerm);
 }
 
-export const generateEndpoints = (parties: number, clientIndex: number, tssEndpoints: string[], wsEndpoints: string[], nodeIndexes: number[]) => {
-  const endpoints = [];
-  const tssWSEndpoints = [];
-  const partyIndexes = [];
+export const generateEndpoints = (tssNodeEndpoints: string[], parties: number, clientIndex: number, nodeIndexes: number[]) => {
```

Comment:
can you outline the changes to this function? maybe a brief function documentation would be nice.

Reply:
previous function was not being used by any apps or sdks because it was hardcoding urls and was only working for sapphire mainnet, so whats happening now that all the sdks and examples ends up implementing their own generateEndpoints function. So in order to avoid the code duplication i changed this function to become exactly same as mpc core kit one as implemented here: https://github.com/Web3Auth/mpc-core-kit/blob/2ae3b41d816a94aabd013c0ceffaf74937e9803d/src/utils.ts#L41

Reply:
This function had hard-coded values in it. #70 corrected this, though it has changed again here. Lets settle this tomorrow.